hubitatmaker/types.py

```diff
@@ -18,6 +18,12 @@ def type(self) -> str:
     def value(self) -> Union[str, float]:
         return self._properties["currentValue"]
 
+    @property
+    def values(self) -> Optional[List[str, ...]]:
```

Comment:
```suggestion
    def values(self) -> Optional[List[str]]:
```
List doesn't use the `,...` notation since lists are assumed to contain elements of the same time.